src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivityToNomisIntTest.kt

```diff
@@ -340,10 +343,12 @@ class ActivityToNomisIntTest : SqsIntegrationTestBase() {
             .withRequestBody(matchingJsonPath("scheduleRules[1].endTime", equalTo("14:25")))
             .withRequestBody(matchingJsonPath("scheduleRules[1].tuesday", equalTo("true")))
             .withRequestBody(matchingJsonPath("scheduleRules[1].thursday", equalTo("false")))
+            .withRequestBody(matchingJsonPath("schedules[0].id", equalTo("$NOMIS_CRS_SCH_ID")))
```

Comment:
When updating a schedule send the Nomis course schedule id found form the mapping API